packages/components/addon/components/hds/form/masked-input/base.hbs

```diff
@@ -2,33 +2,38 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
-<div class={{this.classNames}} {{style width=@width}}>
-  {{#if @isMultiline}}
-    <Hds::Form::Textarea::Base
-      class="hds-form-masked-input__control"
-      @value={{@value}}
-      @isInvalid={{@isInvalid}}
-      @height={{@height}}
-      id={{this.id}}
-      ...attributes
-    />
-  {{else}}
-    <Hds::Form::TextInput::Base
-      class="hds-form-masked-input__control"
-      @value={{@value}}
-      @isInvalid={{@isInvalid}}
-      id={{this.id}}
-      ...attributes
-    />
+<div class="hds-form-masked-input__wrapper">
```

Comment:
FWIW this wrapper is now referred to as `control` in Figma. This is probably a better naming convention in code, just noting the inconsistency here.

Reply:
We have [that wrapper too](https://github.com/hashicorp/design-system/blob/main/packages/components/addon/components/hds/form/field/index.hbs#L27-L29) in code but it sits outside Base, and part of the `Hds::Form::Field` hence we needed a new one 😞

Reply:
@alex-ju I want to rubberduck some ideas on this; I'll set up a meeting for tomorrow

Reply:
Got it, yeah makes sense.